### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Mário André Rocha Ferreira and Kénan Augsburger

### DISCUSSION THREAD

**Discussion by Mondotosz** [heig-vd-dai-course/heig-vd-dai-course#360] [Show and tell]
A CLI tool to compress files with different algorithms such as [RLE](https://en.wikipedia.org/wiki/Run-length_encoding) and [LZW](https://en.wikipedia.org/wiki/Lempel%E2%80%93Ziv%E2%80%93Welch)

The archiving part would use the tar library from apache commons and its output would then be passed through our compression algorithms.

The following commands would be implemented
- compress: Compress files into a single archive with a specified compression algorithm. (allows multiple files)
- deflate: Decompress an archive with the specified algorithm and extract its content.

Members:
- @Mondotosz 
- @Thynkon

TODO:
- [x] GitHub repository link
- [x] Hash of the commit

Repository: [Thynkon/dai-lab-01](https://github.com/Thynkon/dai-lab-01)
Commit: [18c4407](https://github.com/Thynkon/dai-lab-01/commit/18c4407b787c67fbd6903fba2dad96fdadc82ebe)

**Comment by ludelafo** [comment]
All good for us, thank you!
